cocos/2d/renderer/batcher-2d.ts

```diff
@@ -123,11 +123,6 @@ export class Batcher2D implements IBatcher {
         this.device = _root.device;
         this._batches = new CachedArray(64);
         this._drawBatchPool = new Pool(() => new DrawBatch2D(), 128, (obj) => obj.destroy(this));
-
-        if (JSB) {
-            this._nativeObj = new NativeBatcher2d();
-            //this.initAttrBuffer();
-        }
```

Comment:
Why delete it? Then where to create the native object?

Reply:
NativeBatcher2d is allocated in Root::setRenderPipeline and JS visits it through Root::getBatcher2D jsb function.